web-client/src/components/PhoneNumber/PhoneNumber.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, Input, Typography } from 'antd';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import logo from '../../assets/logo.png';
+import TitleWithAddon from '../TitleWithAddon/TitleWithAddon';
+
+const { Text } = Typography;
+
+const StyledIntro = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 50px 50px;
+`;
+
+const Logo = styled.img`
+  height: 125px;
+  width: 125px;
+`;
+
+const Description = styled(Text)`
+  margin-top: 3rem;
+`;
+
+const StyledInput = styled(Input)`
+  margin-top: 1rem;
+  width: 11rem;
+`;
+
+const Info = styled(Text)`
+  color: #ddd;
+  /* margin-top: 40px; */
+  text-align: center;
+`;
+
+const StyledButton = styled(Button)`
+  margin-top: 40px;
+`;
+
+const PhoneNumber: React.FC = (): React.ReactElement => {
+  const { t } = useTranslation();
+
+  return (
+    <StyledIntro>
+      <Logo src={logo} alt="logo" />
+      <TitleWithAddon level={4}>{t('phoneNumber.title')}</TitleWithAddon>
+      <Description>{t('phoneNumber.sub_title')}</Description>
+      <StyledInput placeholder="+0 000 000 000 000" />
+      <Info>{t('phoneNumber.info')}</Info>
+      <StyledButton type="primary">{t('continue')}</StyledButton>
+    </StyledIntro>
+  );
+};
```

Comment:
It could be nice to already have some input handler bindings at this point. You should also be using formik for this kind of situations

Reply:
Hey! 
I think we don't need Formik as a dependency at this point if we are using antd's [Forms](https://ant.design/components/form/). It comes with a way to handle the form submission while abstracting all the validation logic from us (this is customizable). It's really easy to use and avoids the extra dependency. Check this - https://github.com/reach4help/reach4help/blob/screen-new-request/web-client/src/components/NewRequest/NewRequest.tsx

Reply:
Lets use AntD's forms for now. @micaelr95

Reply:
I rewriting the code now